manifests/kots-helm.yaml

```diff
@@ -19,9 +19,9 @@ spec:
       encryptionSecretKey:
         value: '{{repl ConfigOption "autogeneratedEncryptionSecretKey" }}'
       jwtApiSecret:
-        value: '{{repl ConfigOption "autogeneratedVarnishDebugSecret" }}'
-      varnishDebugSecret:
         value: '{{repl ConfigOption "autogeneratedJwtApiSecret" }}'
+      varnishDebugSecret:
```

Comment:
This is a breaking change so should be promoted to prod as quick as possible

Reply:
I'm going to deploy it as soon as it's in main 😄